frontend/src/styles.module.css

```diff
@@ -55,3 +56,20 @@
   display: block;
   margin: auto;
 }
+
+.colorModeSwitch {
+  position: absolute;
+  right: 1rem;
+  bottom: 1rem;
```

Comment:
I find this position to be slightly awkward if the screen is smaller and you scroll to the bottom. Also, it can overlay stuff on smaller screens. Maybe we can integrate it into the headline instead?

Reply:
The issue is that both the header and the footer are not visible on the login screen. Nonetheless, I would like to integrate the switch on that screen as well.

I will try to find a better position that is better suited

Reply:
The thing is, the button is nice but it is not critical, and once you toggled it to your liking, you will likely not touch it ever again. So I think it is not necessary that it is always visible, or at least not always in a prominent position.

Reply:
Implemented a `ColorModeProvider`.